boards/shields/x_nucleo_idb05a1/x_nucleo_idb05a1.overlay

```diff
@@ -5,13 +5,14 @@
  */
 
 &arduino_spi {
-	cs-gpios = <&arduino_header 1 (GPIO_ACTIVE_HIGH | GPIO_PULL_UP)>;	      /* A1 */
+	cs-gpios = <&arduino_header 1 (GPIO_ACTIVE_LOW | GPIO_PULL_UP)>;	      /* A1 */
```

Comment:
Looks fine, but should be a separate commit in the pull request.

Reply:
I agree.